CLASSES/class_person.py

```diff
@@ -0,0 +1,92 @@
+class Person:
+    max_health = 100
+    base_stats = ("attack", "defense", "intelligence", "health")
+
+    def __init__(self, features: dict):
+
+        self.features = features
+        self.first_name = self.features["first_name"]
+        self.last_name = self.features["last_name"]
+        self.full_name = ' '.join([self.first_name, self.last_name])
+
+        self.occupation = self.features["occupation"]
+        self.health = self.max_health
```

Comment:
Możesz sobie dodać trochę elastyczności biorąc tą wartość ze słownika. Żeby się nie powtarzać to możesz zapisywać tylko zdrowie bohatera, który ma mniej niż 100
self.health = features.get('health', self.max_health) -> czyli jak nie ma klucza health to ustaw 100(max_heath)